README.md

```diff
@@ -64,3 +64,9 @@ brew install pantheon-systems/external/terminus
 ### Other installation methods
 
 Refer to the [Terminus manual](https://pantheon.io/docs/terminus/install#install-terminus) for other installation methods.
+
+
+## NEW RELEASE PROCESS
+
+1. Merge an approved PR into the `3.x` branch. A new minor version will be tagged.
```

Comment:
This should tag a dev version, not a minor version. Minor versions are stable.

Reply:
Merging increments the version by one in the version file. During the build process, you're always building (.Version+1)-dev. On release, you're building (.version)